client/src/components/ChangePassword.jsx

```diff
@@ -17,7 +16,7 @@ export class ChangePassword extends Component {
 
     change_old_password = (e) => {
         if (this.state.new_password !== this.state.check_new_password) {
-            window.Materialize.toast("Wrong field confirm password ", 3000);
+            window.Materialize.toast("Passwords do not match ", 3000);
             return 1;
         }
         let change_password_data = {
```

Comment:
If you don't change the variable better to use `const`.

Reply:
Where?

Reply:
Here